doc/ref/gappkg.xml

```diff
@@ -173,5 +187,1995 @@ users.
+
+The filename may optionally have an extension, e.g. <F>.txt</F> or <F>.md</F>.
+<P/>
+This should contain <Q>how to get it</Q> instructions (covering the 
+way of getting it with the &GAP; distribution and from the &GAP; website, 
+if applicable), as well as installation instructions and names
+of the package  authors and their email addresses. These installation
+instructions should be repeated or referenced from the package's 
+documentation, which  should be in the <F>doc</F> directory
+(see <Ref Sect="Writing Documentation and Tools Needed"/>).
+Authors' names and addresses should be repeated both in the package's 
+documentation and in the <File>PackageInfo.g</File> (see below).
+</Item>
+
+<Mark>
+<F>CHANGES</F></Mark>
+<Item>
+For further versions of the package, it will be also useful to have a 
+<F>CHANGES</F> file that records the main changes between versions 
+of the package.
+<P/>
+The filename may optionally have an extension, e.g. <F>.txt</F> or <F>.md</F>.
+</Item>
+
+<Mark>
+<F>LICENSE</F></Mark>
+<Item>
+The file which explains conditions on which the package is distributed.
+<P/>
+We advise all package authors to make clear in the documentation of their
+package the basis on which it is being distributed to users. Technically,
+this is the terms of the license which you give the users to copy, modify
+and redistribute your software (of which you presumably own the copyright)
+for their purposes.
+<P/>
+&GAP; itself is distributed under the GNU General Public License version 2,
+a popular <Q>free software</Q> license which allows users to redistribute it
+freely under the same terms, and requires that any software which
+incorporates &GAP; (technically, any <Q>derived work</Q>) also be distributed
+under those terms. We would encourage you to consider the GPL for your
+packages, but you might wish to be more restrictive (for instance
+forbidding redistribution for profit) or less restrictive (allowing
+your software to be incorporated into commercial software).
+<P/>
+The filename may optionally have an extension, e.g. <F>.txt</F> or <F>.md</F>.
+Some packages also use different filenames, like <F>COPYING</F>.
+</Item>
+
+<Mark><F>configure</F>, <F>Makefile.in</F></Mark>
+<Item>
+These files are typically only used by packages which have a non-&GAP; component,
+e.g.&nbsp;some C code (the files of which should be in the <F>src</F>
+directory). The <F>configure</F> and <F>Makefile.in</F> files of the 
+<Package>Example</Package> package provide prototypes (or they may be created 
+using the <Package>PackageMaker</Package> mentioned above). The <F>configure</F>
+file typically takes a path <A>path</A> to the &GAP; root directory as argument
+and uses the value assigned to <C>GAParch</C> in the file <F>sysinfo.gap</F>,
+created when &GAP; was compiled to determine the
+compilation architecture, inserts this in place of the string <C>@GAPARCH@</C>
+in <F>Makefile.in</F> and creates a file <F>Makefile</F>. When <C>make</C> is
```

Comment:
This information is outdated with the new build sys. While this still works (due to the "compatibility mode"), the plan is for `sysinfo.gap` to go away, or at last change substantially. I am not comfortable telling people to use it this way. :/

That said, right now the alternatives are not yet ready, so we may have no choice. I'll try to work on the package build system side next week, but unless I make progress, we'll just have to merge this, but I must make a big note somewhere to remind me that this piece of text exists and needs to be updated.

Reply:
I've put the comment in the XML source about this.